MessageLogger/src/main/java/io/github/juby210/acplugins/messagelogger/ReAdder.java

```diff
@@ -63,7 +63,7 @@ public List<Message> reAddDeletedMessages(List<Message> messages, Long channelId
         reAddIDs.sort(comparator);
         if (!(messages instanceof ArrayList)) messages = new ArrayList<>(messages);
         var len = reAddIDs.size();
-        for (var i = 0; i < len; i++) {
+        for (var i = 0; i < len - savedIDs.size(); i++) {
```

Comment:
Readding messages doesn't work because condition is always i < 0
![image](https://user-images.githubusercontent.com/31005896/189539838-23979e60-c2bb-438c-9dec-f1ea9df57353.png)

Reply:
> Readding messages doesn't work because condition is always i < 0 ![image](https://user-images.githubusercontent.com/31005896/189539838-23979e60-c2bb-438c-9dec-f1ea9df57353.png)

oh shit im probably just dumb then